### DISCUSSION TITLE
Prompt Library Export/Backup

### DISCUSSION THREAD

**Discussion by xzlee3** [zed-industries/zed#14559] [Feature Requests]
Is it possible to do the above? Otherwise would have to manually insert the prompts on every device. I see that there's `prompts-library-db.0.mdb` in the config, does a manual sync of the folder work?

**Comment by lunelson** [comment]
👍🏼 to this. IMO the addition of Zed AI, with the prompt library and the context history, begs even more the question of some kind of cloud backup: these things are no longer merely settings, but assets that are continuously developed; they need backing up just like code

**Reply by jakubjezek001** [reply]
> 👍🏼 to this. IMO the addition of Zed AI, with the prompt library and the context history, begs even more the question of some kind of cloud backup: these things are no longer merely settings, but assets that are continuously developed; they need backing up just like code

Agree on this. I am just facing the issue. And just simple copy of .mdb files over to another machine does not work. But yeah, perhaps packing it up over Github gists would be enough, right?

**Comment by xzlee3** [comment]
Any updates on this?

**Comment by mcobzarenco** [comment]
Why aren't prompts just a bunch of text files in your `.config/zed` directory? I found it surprising this is not the case

**Reply by a3lem** [reply]
> Why aren't prompts just a bunch of text files in your `.config/zed` directory? I found it surprising this is not the case

Had the exact same thought today. Global prompts could go there, while project-specific prompts can be kept under the project root. That way, everyone in the team that's using Zed can re-use each other's prompts.

While I don't use Cursor, I think it's impressive how their community has embraced `.cursorrules` (which is limited to system prompt content). So I nominate `.zedprompts.toml` for Zed.

**Reply by lpsm-dev** [reply]
This is an extremely essential feature. Currently it's a pain to manage these prompts manually...

I can't see any zed settings that helped me put these prompts in one place and reuse them.

**Reply by KieranP** [reply]
I'd like to see this alongside project level settings which zed already supports.

e.g. within project root:

```
.zed/settings.json
.zed/default_prompt.mds
```

**Comment by rubiojr** [comment]
I was looking for a way to export and version control my prompts today, so created a quick tool that may be of use while we wait for something official: https://github.com/rubiojr/zed-prompts

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
